src/main/java/uk/co/ramp/covid/simulation/DailyStats.java

```diff
@@ -321,7 +321,7 @@ public void incInfectionsHomeInhabitant() {
 
     public void determineRValues(Population p) {
         RStats rs = new RStats(p);
-        R = rs.getMeanR(day);
+        secInfections = rs.getMeanR(day);
```

Comment:
Slightly confusing. Should you rename getMeanR to getSecInfections ?

Reply:
I think this function is used for 2 things based on a flag. I think it might be neater if we just have 2 different functions and put the flag here?, e.g. if(rflag) { getMeanR } else { getSecondaryInfections } - just for clarity?